packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -195,6 +195,82 @@ enum AnimationBehavior {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
+///
```

Comment:
There should be an explanation here telling the reader what this sample code shows.

Reply:
Also: There's already a lot of example code on this class. What is this showing that's not already covered by the other examples?

Reply:
Is it possible to convert the existing samples here into dartpads?